platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFile.java

```diff
@@ -56,4 +62,49 @@ public Path getConfigFilePath() {
     }
 
     public abstract void createConfig() throws ConfigurationFileException;
+
```

Comment:
Good point @brendan-hofmann. The `write()` wasn't used before and that code was more like a placeholder. We do need to update this to use a try-with-resource to make sure the stream always gets closed.